scripts/check-links.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Check links in the given files don't 404.
```

Comment:
Does it follow redirects? Can we specify it here please?

Reply:
yes can add a comment that it does follow